lib/hammer_cli_foreman/commands.rb

```diff
@@ -279,6 +279,86 @@ def should_retrieve_all?
 
   end
 
+  class ListSearchCommand < ListCommand
```

Comment:
Could we have some more descriptive name? It took me while until I figured out what this class does. Would make `AssociatedResourceListCommand` more sense?